x-pack/plugin/inference/qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/xpack/application/OpenAiServiceUpgradeIT.java

```diff
@@ -120,6 +120,7 @@ public void testOpenAiCompletions() throws IOException {
         final String upgradedClusterId = "upgraded-cluster-completions";
 
         if (isOldCluster()) {
+            // TODO why is put only in old cluster?
```

Comment:
```suggestion
```

There is another endpoint put in the upgraded cluster at line 148. However, there is no put in the mixed cluster, is that what you are referring to?